packages/edit-site/src/components/page-pages/index.js

```diff
@@ -222,7 +222,7 @@ export default function PagePages() {
 						<VStack spacing={ 1 }>
 							<View
 								as="span"
-								className="edit-site-page-pages__list-view-title-field"
+								className="dataviews-view-grid__title-field"
```

Comment:
This makes the `dataviews-view-grid__title-field` a public API. We need dataviews to handle it based on what's the primary field instead.